base_layer/core/src/base_node/comms_interface/inbound_handlers.rs

```diff
@@ -38,8 +38,10 @@ use crate::{
     mempool::{async_mempool, Mempool},
     transactions::transaction::{TransactionKernel, TransactionOutput},
 };
+use async_std::sync::RwLock;
```

Comment:
https://docs.rs/tokio/0.2.13/tokio/sync/struct.RwLock.html

Reply:
The tokio version sems to run better across systems
https://www.reddit.com/r/rust/comments/f4zldz/i_audited_3_different_implementation_of_async/

Reply:
Thank you, I was able to replace the async-std async RwLock with the tokio sync RwLock.